bindings/zig/src/opendal.zig

```diff
@@ -0,0 +1,74 @@
+pub const opendal = @cImport(@cInclude("opendal.h"));
```

Comment:
Hi, can you demonstrate how users will use opendal-zig? Do they need to include opendal.h too? How can we provide something like a zig package so that users can use them like `@import("opendal")`?

Reply:
Currently `zig binding` will depend on `c binding` to work.
The purpose is to unify any changes between the two. That is, it means that with every change in the C header, all you have to do is rebuild the `zig binding` and it will change the `cimport.zig` too.

`@import("opendal")` == `opendal.zig` to zig project, use module on your project `build.zig`.
https://github.com/apache/incubator-opendal/blob/b25a32152508df6b98481d504114d1207f351db4/bindings/zig/build.zig#L7-L14

More info (my post): https://kassane.github.io/2023/05/03/zig-pkg/

Reply:
Currently, zig module equivalence to header-only on C/C++. Library only `opendal_c` + os-libc!!

Reply:
Thanks for the explanation!